### ISSUE TITLE
Option to remove ugly square brackets

### ISSUE THREAD

**Issue by steelwil** [TaoK/PoorMansTSqlFormatter#133]
We do not use spaces or keywords in our names I prefer that they all be removed.

It would however be nice to include such checks and only have square brackets where they should be.

Thanks again for your great tool.

**Comment by kenny-evitt** [comment]
I've got a very small library with code to determine whether an identifier is 'regular' (i.e. does not need to be delimited):

 - [kenny-evitt/Transact-SQL-Helpers](https://github.com/kenny-evitt/Transact-SQL-Helpers)

Here's the relevant class:

 - [Transact-SQL-Helpers/Identifiers.cs at master · kenny-evitt/Transact-SQL-Helpers](https://github.com/kenny-evitt/Transact-SQL-Helpers/blob/master/TransactSqlHelpers/Identifiers.cs)

Basically, instead of delimiting all identifiers you could output the value of  `Identifiers.ValidIdentifier(identifierString)` instead. If an identifier is 'regular', i.e. of the expected format and not a reserved keyword, that method will just output the identifier as-is; otherwise it outputs the identifier delimited.
